### DISCUSSION TITLE
Unordered List

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#229] [Component suggestions]
<img width="480" alt="visuele weergave van het unordered list component" src="https://github.com/nl-design-system/backlog/assets/545207/86c48b0f-b61c-44d7-b017-4ce99190d5f0">

We willen graag meer leren over het Unordered List component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1710341350557159)

---

## Naam
Unordered List

## Doel
Opsomming waar de volgorde van items in de lijst niet van belang is.

## Gerelateerde componenten
[Ordered List](https://github.com/orgs/nl-design-system/discussions/230), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289), [Link List](https://github.com/orgs/nl-design-system/discussions/266), [Task List](https://github.com/orgs/nl-design-system/discussions/295)

## Onderzoek

[7 Tips for Presenting Bulleted Lists in Digital Content - Nielsen Norman Group](https://www.nngroup.com/articles/presenting-bulleted-lists/)
Artikel met tips voor het presenteren van Unordered List.

**Comment by jeffreylauwers** [comment]
## Amsterdam

- [Storybook](https://amsterdam.github.io/design-system/?path=/docs/components-text-unordered-list--docs)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/unordered-list)
- [Figma](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?type=design&node-id=5003-19229&mode=design&t=fR3GEi2CGJzOMP9C-4)

---

### Screenshots / Varianten

**Storybook**

Default
Items of an unordered list have square markers. There is extra white space between the items. This provides better distinction between the items, especially when they consist of multiple lines of text.
![Scherm­afbeelding 2024-03-14 om 10 58 36](https://github.com/nl-design-system/backlog/assets/545207/0e1d8beb-5a5a-4c02-9431-90df47c1f301)

Two Levels
A list can have one deeper level. In this case, an en dash is used as the list item marker. It is left-aligned with the text of the higher-level item.
![Scherm­afbeelding 2024-03-14 om 10 58 47](https://github.com/nl-design-system/backlog/assets/545207/71d55892-e289-4860-be47-ec070e857458)

Inverse Colour
Set the inverseColor prop if the List sits on a dark background. This ensures the colour of the text provides enough contrast. When nesting lists, set the prop on all lists.
![Scherm­afbeelding 2024-03-14 om 10 59 24](https://github.com/nl-design-system/backlog/assets/545207/3c396c26-a85b-4401-93de-377633a13746)

Without Markers
An overview of equivalent elements is also an unordered list. In such cases, apply the list semantics while hiding markers. This helps screen readers understand and navigate the list.

Note: In this variant, each item typically contains a container with titles and other text components. Therefore, this variant does not define typography for the items and the inverseColor prop has no effect.

This example is based on the top tasks on the homepage of the website:
![Scherm­afbeelding 2024-03-14 om 11 39 07](https://github.com/nl-design-system/backlog/assets/545207/dd51174c-1fbc-4261-b769-941315ec8163)



**Figma**
![Scherm­afbeelding 2024-03-14 om 11 05 31](https://github.com/nl-design-system/backlog/assets/545207/454c7e7b-6161-4b2a-9c61-f52e5e94c3ce)

---

### Notities
- Storybook component biedt de mogelijkheid om de kleur om te keren bij een donkere achtergrond.
- In Figma wordt het component gecombineerd met de ordered list.
- In Figma de optie om een groen check of rood cross icoon te tonen in plaats van een bullet.
- Documentatie beschikbaar (Storybook).

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-unorderedlist--docs)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/UnorderedList)
- [Figma - Huidig](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?type=design&node-id=8105-22880&mode=design&t=TDQARyHiaCmjMM77-4)
- [Figma - WIP](https://www.figma.com/file/x4RkF6BIdrNZbh7D53NTzB/%F0%9F%92%A0-NLDS---Den-Haag---Bibliotheek?type=design&node-id=828-934&mode=design&t=4NP6wIn0FtjvCEOS-4)

---

### Screenshots / Varianten

**Storybook**

Unordered List with nesting
![Scherm­afbeelding 2024-03-14 om 11 15 42](https://github.com/nl-design-system/backlog/assets/545207/8c632132-bc66-4e25-8520-cc58c5843209)

Unordered list + Paragraph
![Scherm­afbeelding 2024-03-14 om 11 15 56](https://github.com/nl-design-system/backlog/assets/545207/54fedc74-4019-456c-ae20-f5811a318bca)

Figma - Huidig
![Scherm­afbeelding 2024-03-14 om 11 17 18](https://github.com/nl-design-system/backlog/assets/545207/fed479cc-85f3-4a39-a37f-c51f477745ca)

Figma - WIP
![Scherm­afbeelding 2024-03-14 om 11 18 18](https://github.com/nl-design-system/backlog/assets/545207/6d171df0-aa92-4fdc-9275-867a3b6127df)

---

### Notities
- Documentatie beschikbaar.

**Reply by Rozerinay** [reply]
Opmerking afkomstig uit een comment in de NL Design System Figma bibliotheek:

Token `utrecht.unordered-list.item.margin-block-start` en `utrecht.unordered-list.item.margin-block-end` zijn in Figma gebruikt als padding, en niet opgenomen in het voorbeeldthema (JSON). Mogelijk vervangen we dit later door `row-gap` voor een duidelijkere scheiding tussen items.

**Comment by jeffreylauwers** [comment]
## DSO

[Storybook](https://dso-toolkit.nl/62.8.4/components/lists/list)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-14 om 11 29 59](https://github.com/nl-design-system/backlog/assets/545207/472ab83b-e696-4c1d-9ab8-da27ed9079aa)

Group
![Scherm­afbeelding 2024-03-14 om 11 30 08](https://github.com/nl-design-system/backlog/assets/545207/6d38bc00-6583-4e34-9f93-5a9de3cc7dff)

Columns
![Scherm­afbeelding 2024-03-14 om 11 30 16](https://github.com/nl-design-system/backlog/assets/545207/51ebf3c1-20cf-4754-9d3c-2228423b7477)

Image list
![Scherm­afbeelding 2024-03-14 om 11 30 23](https://github.com/nl-design-system/backlog/assets/545207/53fb0397-749c-467e-99ac-5857bf7a60f7)

Unstyled
![Scherm­afbeelding 2024-03-14 om 11 32 16](https://github.com/nl-design-system/backlog/assets/545207/5becd8f0-26d9-447c-8937-58ca53b87087)

Icons
![Scherm­afbeelding 2024-03-14 om 11 31 50](https://github.com/nl-design-system/backlog/assets/545207/356027ff-d9c5-4721-a083-cd61c4146599)

---

### Notities
- Ondergebracht bij [List](https://storybook.dso-toolkit.nl/62.8.4/?path=/docs/html-css-list--docs).

**Comment by jeffreylauwers** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/lists)

---

### Screenshots / Varianten

**Documentatie website**

Default
![Scherm­afbeelding 2024-03-14 om 11 40 37](https://github.com/nl-design-system/backlog/assets/545207/66041cd6-c15a-45c2-b426-a27d6b7c728e)

Genest lijst
![Scherm­afbeelding 2024-03-14 om 11 40 56](https://github.com/nl-design-system/backlog/assets/545207/1db7e7bd-5e06-441c-a36f-222abae08c9f)

Dash
![Scherm­afbeelding 2024-03-14 om 11 41 01](https://github.com/nl-design-system/backlog/assets/545207/58f95e96-6b8f-4497-ad54-62c417cb5f42)

Link lijst
![Scherm­afbeelding 2024-03-14 om 11 41 07](https://github.com/nl-design-system/backlog/assets/545207/eeaec4dc-8c6f-4c00-a1fc-8c9c7859895c)

Link lijst met chevron
![Scherm­afbeelding 2024-03-14 om 11 41 15](https://github.com/nl-design-system/backlog/assets/545207/c50202ed-1510-4d81-8722-759702474df6)

Link lijst met chevron op een achtergrond
![Scherm­afbeelding 2024-03-14 om 11 41 25](https://github.com/nl-design-system/backlog/assets/545207/900b1b77-ef7e-4e5d-a293-4139d254d956)

Onderwerpen
![Scherm­afbeelding 2024-03-14 om 11 41 30](https://github.com/nl-design-system/backlog/assets/545207/4808647d-006d-4a9f-9c77-277ee4a93618)

---

### Notities
- Ondergebracht bij Lists.
- Twee varianten voor [Ordered list](https://github.com/orgs/nl-design-system/discussions/230#discussioncomment-8785802).

**Comment by jeffreylauwers** [comment]
## Open Gemeenten

- [Documentatie website - Standaardpagina](https://componenten.lenteveld.nl/paginatype-standaard)
- [Patternlab](https://patternlab.lenteveld.nl/index.html)

---

### Screenshots / Varianten

**Documentatie website**

![Scherm­afbeelding 2024-03-14 om 11 53 34](https://github.com/nl-design-system/backlog/assets/545207/2136ecae-d2d9-45cd-8002-ac5628d477aa)


---

### Notities
- Niet kunnen vinden in Patternlab. Vandaar screenshot documentatie omgeving. Vraag staat uit bij Martijn.

**Comment by jeffreylauwers** [comment]
## Rotterdam

- [Storybook](https://nl-design-system.github.io/rotterdam/?path=/docs/css-utrecht-unordered-list--docs)
- [Figma](https://www.figma.com/file/ZWSC4gCrOXRUR9UX3aoZ8x/Gemeente-Rotterdam-Design-System?type=design&node-id=490-17441&mode=design&t=BL7EnBsYrSwV4sCp-4)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-14 om 11 56 06](https://github.com/nl-design-system/backlog/assets/545207/0cd4dcd2-ab5c-451a-8384-a4848cdb5c96)

**Figma**

Bullet lists
![Scherm­afbeelding 2024-03-14 om 11 56 35](https://github.com/nl-design-system/backlog/assets/545207/332ef82d-cbe4-4d22-a908-bc2059d9a945)

Link lists with icon
![Scherm­afbeelding 2024-03-14 om 11 56 48](https://github.com/nl-design-system/backlog/assets/545207/b8e09b88-2924-4877-8c51-4361c7e48da2)

Text button lists
![Scherm­afbeelding 2024-03-14 om 11 56 56](https://github.com/nl-design-system/backlog/assets/545207/2c7bb17b-7931-41ef-8b1d-749931b7a360)

---

### Notities

**Comment by jeffreylauwers** [comment]
## RVO

- [Documentatie website](https://nl-design-system.github.io/rvo/docs/components/ordered-unordered-list/)
- [Storybook](https://nl-design-system.github.io/rvo/?path=/docs/componenten-ordered-unordered-list--documentatie)
- [GitHub](https://github.com/nl-design-system/rvo/tree/master/components/ordered-unordered-list)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-14 om 12 05 54](https://github.com/nl-design-system/backlog/assets/545207/1c368899-3e59-4559-88c1-8e2130fd1360)

Icon
![Scherm­afbeelding 2024-03-14 om 12 06 13](https://github.com/nl-design-system/backlog/assets/545207/0687ff4e-393e-4a73-a2b5-44807200a3bc)

None
![Scherm­afbeelding 2024-03-14 om 12 06 22](https://github.com/nl-design-system/backlog/assets/545207/31b220bf-c695-421b-97ed-d98a7ebd82e5)

---

### Notities
- Samengevoegd met [ordered list](https://github.com/orgs/nl-design-system/discussions/230#discussioncomment-8785896).
- Voor bullet type 'Icon' 3 varianten beschikbaar.

**Comment by jeffreylauwers** [comment]
## Utrecht

- [Documentatie website](https://nl-design-system.github.io/utrecht/docs/componenten/data%20display/unordered-list)
- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/html_html-unordered-list--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/unordered-list)
- [Figma](https://www.figma.com/file/UXIHcIurAD8hyoBWx4hDBV/NLDS---Gemeente-Utrecht---Bibliotheek?type=design&node-id=828-934&mode=design&t=Hjc3exjf4jE1oDqm-4)

---

### Screenshots / Varianten

**Storybook**
Default
![Scherm­afbeelding 2024-03-14 om 12 10 58](https://github.com/nl-design-system/backlog/assets/545207/6f649202-54ef-4c4d-9d70-c32b7b92fcab)

Nested
![Scherm­afbeelding 2024-03-14 om 12 11 03](https://github.com/nl-design-system/backlog/assets/545207/218adbf1-b6fb-4c58-8202-b31afd320c18)

**Figma**
![Scherm­afbeelding 2024-03-14 om 12 11 47](https://github.com/nl-design-system/backlog/assets/545207/4e25a749-60bb-4870-b992-29cea3c307c1)


---

### Notities
- Documentatie website bevat richtlijnen.
- Token `utrecht.unordered-list.item.margin-block-start` en `utrecht.unordered-list.item.margin-block-end` zijn in Figma gebruikt als padding, en niet opgenomen in het voorbeeldthema (JSON). Mogelijk vervangen we dit later door `row-gap` voor een duidelijkere scheiding tussen items.

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Unordered list op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=unordered-list)

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Unordered list op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=unordered+list)

**Comment by JeroenduC** [comment]
Input tijdens Design Open Dag (30-1): 
- Wil je meer dan twee levels ondersteunen in je nesting? Wanneer gaat het tegen zich werken. Richtlijn om toe te voegen aan het component?

**Comment by pixelgitter** [comment]
Bij utrecht maken wij binnenkort gebruik van verschillende markers als we nesting lists gebruiken. 

Deze markers zullen instelbaar zijn via de tokens. Wij gebruiken `::marker` om een leesteken in de `content` parameter in te stellen, en een `font-size` om de grootte ervan te bepalen. De reden om de grootte ook instelbaar te hebben is dat het kan voorkomen dat er visueel inconsistentie kan ontstaan bij de verschillende tekens. Mogelijk verschilt dit ook per font-family. Een ▲ kan kleiner uitvallen vergeleken een ●. 

<img width="820" height="452" alt="image" src="https://github.com/user-attachments/assets/d1b7d745-652e-46b4-a241-c300e4259355" />

De keuze van de verschillende markers zijn op basis van de Apple Notitie app:

<img width="506" height="340" alt="image" src="https://github.com/user-attachments/assets/005da23e-7b98-4bdf-8bd2-9d967391a0e2" />

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al ingesteld voor meerdere huisstijlen?

Het kernteam heeft gekeken of Unordered List van de Gemeente Utrecht al geschikt is om richting Hall of Fame te gaan. Hiervoor willen we dat de component succesvol in tenminste 2 verschillende huisstijlen is ingezet.

[We hebben gekeken naar de huisstijl van onder andere](https://github.com/nl-design-system/candidate/issues/1070#issuecomment-4031455046) de gemeente Hoorn, Noordwijk en implementaties van OpenStad en we zijn tot de volgende conclusie gekomen:

🎉 Jazeker! Met jullie hulp hebben we genoeg informatie verzameld om een goed instelbaar component als Candidate beschikbaar te gaan maken.

**Comment by JeroenduC** [comment]
Hoera! 😄
________________________________
Van: Renate Roke ***@***.***>
Verzonden: dinsdag 10 maart 2026 15:37
Aan: nl-design-system/backlog ***@***.***>
CC: Chatinier, Jeroen du ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [nl-design-system/backlog] Unordered List (Discussion #229)


Candidate selectieproces: Is de component al ingesteld voor meerdere huisstijlen?

Het kernteam heeft gekeken of Unordered List van de Gemeente Utrecht al geschikt is om richting Hall of Fame te gaan. Hiervoor willen we dat de component succesvol in tenminste 2 verschillende huisstijlen is ingezet.

We hebben gekeken naar de huisstijl van onder andere de gemeente Hoorn, Noordwijk en implementaties van OpenStad en we zijn tot de volgende conclusie gekomen:

🎉 Jazeker! Met jullie hulp hebben we genoeg informatie verzameld om een goed instelbaar component als Candidate beschikbaar te gaan maken.

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al ingezet in meerdere projecten?

Omdat de Unordered List van Gemeente Utrecht geschikt is voor verschillende huisstijlen heeft [het kernteam ook gekeken of hij al gebruikt wordt in productie, prototypes en templates](https://github.com/nl-design-system/candidate/issues/1071#issuecomment-4045089074).
Deze informatie is nodig om de component een stap verder te brengen in het Candidate proces.

We hebben hiervoor op verschillende websites gekeken waar de component succesvol is ingezet en we zijn tot de volgende conclusie gekomen:

🎉 De component wordt onder andere gebruikt door:

- Gemeente Utrecht gebruikt het in [Digitaal loket ](https://loket.digitaal.utrecht.nl/nl/products/verhuizing-doorgeven)
- OpenStad gebruikt het op de website van [OpenStad ](https://openstad.org/over-openstad/hoe-het-werkt)
- Daarnaast is de component op diverse websites via OpenStad ingezet, zoals op:
 - [Doe Mee Druten Wijchen](https://doemee.drutenwijchen.nl/zon-wind/) 
 - [Buurtwarmtenet Watergraafsmeer ](https://buurtwarmtenet-watergraafsmeer.nl/startcontract)
 - [Denk Mee D66 ](https://denkmee.d66.nl/)
 - [Oost aan Zet ](https://oostaanzet.nl/)
 
 Heb je aanvullingen? We horen het graag hier in de discussions!

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al meegenomen in een WCAG-EM audit?

Om de Unordered List van de Gemeente Utrecht als een toegankelijk Candidate component beschikbaar te maken is het handig als we weten of de Community implementaties al helemaal toegankelijk kunnen worden ingezet.

We hebben hiervoor [gekeken of er een toegankelijkheidsverklaring beschikbaar was voor de projecten die in productie worden gebruikt](https://github.com/nl-design-system/candidate/issues/1072#issuecomment-4045634553). En:

🎉 Ja! We hebben de uitkomsten van de volgende WCAG-EM audit (1 of meerdere) kunnen vinden:

- [Digitaal Loket Gemeente Utrecht](https://toegankelijkheidsrapport.frameless.io/rapport/loket.digitaal.utrecht.nl-2024-06) 
- [NL Design System ](https://nldesignsystem.nl/toegankelijkheidsverklaring/rapport-2024/)
- [OpenStad techniek ](https://toegankelijkheidsrapport.swink.nl/openstad-hercontrole/audit/)
- [OpenStad content Druten Wijchen ](https://www.wijchen.nl/data/downloadables/1/8/9/0/wcag-22-aa-deelonderzoek-content-doemeedrutenwijchennl-10-202503141111.pdf)

Heb je aanvullingen? We horen het graag hier in de discussions!

**Comment by eslook** [comment]
## Candidate voorbereidingsfase: Issues bekend

We hebben gekeken bij Amsterdam, RVO, Utrecht, Rotterdam, Tilburg, Den Haag en Rijkshuisstijl Community naar openstaande issues en pull request gerelateerd aan de Unordered List, de input uit de GitHub discussion meegenomen en issues bekend bij de community uitgevraagd. Daaruit zijn de volgende issues bekend gekomen.

We hebben besloten om deze issues wel mee te nemen, om de volgende redenen:

- [Utrecht](https://github.com/nl-design-system/utrecht)
  - [Ordered list has fallback tokens to unordered list, doesn't seem right](https://github.com/nl-design-system/utrecht/issues/2426)
    Dit is relevant omdat componenten niet van elkaars tokens gebruik moeten maken. Hier houden we standaard rekening mee.
  - [Ordered and unordered list have no color tokens](https://github.com/nl-design-system/utrecht/issues/2349)
    Dit is relevant, we hebben meegenomen in het Candidate proces dat de Ordered List en de Unordered List allebei een color token krijgen.
  - [Avoid large margin for nested lists in rich text](https://github.com/nl-design-system/utrecht/issues/2700)
    Dit is relevant omdat componenten rekening moeten houden met de spacing van componenten ervoor en erna. Hier houden we standaard rekening mee.
  - [docs: forced colors story for CSS unordered list](https://github.com/nl-design-system/utrecht/pull/2506)
    Dit is relevant omdat we altijd een forced color story willen meenemen voor een Candidate component. Hier houden we standaard rekening mee.
  - [feat: updated unordered-list to use SVG markers](https://github.com/nl-design-system/utrecht/pull/2398) 
  	Dit is relevant, en we nemen in de component mee dat de markers vervangen kunnen worden met SVG.
- [Den Haag](https://github.com/nl-design-system/denhaag)
  - [Spacing between components](https://github.com/nl-design-system/denhaag/issues/1121)
  	Dit is relevant omdat componenten rekening moeten houden met de spacing van componenten ervoor en erna. Hier houden we standaard rekening mee.
- [Uit de GitHub discussie](https://github.com/orgs/nl-design-system/discussions/229)
  - DSO heeft ook de versie `columns` waarvoor we een Story meenemen in Test Storybook.
  - Uit Design Open Dag van 30 januari 2026 kwam de vraag naar richtlijnen voor het aantal levels om te ondersteunen, en te gebruiken. Om deze vraag te beantwoorden voor Candidate hebben we een onderzoek uitgevoerd.

We hebben besloten om deze issues niet mee te nemen, om de volgende redenen:


- [RVO](https://github.com/nl-design-system/rvo)
  - [Use existing NL Design System community React components where possible](https://github.com/nl-design-system/rvo/issues/265)
    Deze issue is overkoepelend, en het deel relevant voor deze component is al voltooid.
- [Utrecht](https://github.com/nl-design-system/utrecht)
  - [Component: Unordered list](https://github.com/nl-design-system/utrecht/issues/24)
    Deze issue ging om het implementeren van de component. De component is al geimplementeerd. We hebben de issue gesloten.
  - [alle Angular componenten getest](https://github.com/nl-design-system/utrecht/issues/1589)
    Deze issue gaat om Angular tests toevoegen in Utrecht.
  - [alle Angular componenten in Storybook](https://github.com/nl-design-system/utrecht/issues/1588)
    Deze issue gaat om Angular stories toevoegen in Utrecht.
  - [React component library aanbieden](https://github.com/nl-design-system/utrecht/issues/833)
    Deze issue is overkoepelend, en het deel relevant voor deze component is al voltooid.
  - [Meer componenten beschikbaar maken voor Vue 3](https://github.com/nl-design-system/utrecht/issues/1688)
    Deze issue gaat om een Vue implementatie in Utrecht.
  - [feat: ordered and unordered list item web component](https://github.com/nl-design-system/utrecht/pull/2013)
    Deze issue gaat om een Web Component implementatie in Utrecht.
- [Den Haag](https://github.com/nl-design-system/denhaag)
  - [Improve and extend Utrecht component design tokens](https://github.com/nl-design-system/denhaag/issues/1081)
    Deze issue is overkoepelend, en het deel relevant voor deze component is al voltooid.
  - [UnorderedList](https://github.com/nl-design-system/denhaag/issues/1459)
    Dit is specifiek voor Den Haag met hun branches en heeft geen impact op Candidate.
- [Rijkshuisstijl Community](https://github.com/nl-design-system/rijkshuisstijl-community)
  - [Verbeteren: Unordered list component](https://github.com/nl-design-system/rijkshuisstijl-community/issues/973)
  	Deze issue is verlaten, en het laatste wat nog niet uitgevoerd was voor de component, was de documentatie controleren.
  - [Componenten in Storybook met nieuwe Rijkshuisstijl opleveren](https://github.com/nl-design-system/rijkshuisstijl-community/issues/2316)
  	Deze issue gaat specifiek om het nieuwe Rijkshuisstijl design toe te passen in Rijkshuisstijl Community.
  - [Tracking Issue: Twig support voor Drupal MVP](https://github.com/nl-design-system/rijkshuisstijl-community/issues/1307)
  	Deze issue gaat om een Twig implementatie in Rijkshuisstijl Community.
  - [Moved documentation of components Table up to Unordered List into documentation/components folder](https://github.com/nl-design-system/rijkshuisstijl-community/pull/1987)
	Deze issue gaat om het verplaatsen van bestanden binnen de Rijkshuisstijl Community.
- [GitHub discussie](https://github.com/orgs/nl-design-system/discussions/229).
  - De comment van Den Haag over de Utrecht Unordered List tokens in Figma zijn niet relevant; dit gaan ze zelf oplossen en gaat in NL Design System al goed.


Uit [Amsterdam](https://github.com/amsterdam/design-system) en [Rotterdam](https://github.com/nl-design-system/rotterdam) zijn geen issues bekend gekomen.

Bekijk in [deze Candidate issue](https://github.com/nl-design-system/candidate/issues/1074) hoe we tot dit resultaat zijn gekomen.

### Hoe kun je helpen om deze features wel in de Candidate implementatie te krijgen?

Pas de wijzigingen eerst in de Community implementatie toe en vraag andere organisaties in de community om deze ook in hun projecten te gebruiken. Documenteer jullie bevindingen en successen in de GitHub Discussion, deel deze in de Heartbeat en op Slack. Zo heeft het kernteam meer bewijs dat de feature wenselijk en haalbaar is en kan deze besluiten de functionaliteit alsnog, of in een volgende versie mee te nemen.

**Comment by Rozerinay** [comment]
## Candidate voorbereidingsfase: Versimpeld of gesplitst

De varianten voor de Candidate component zijn bepaald, dat hebben we gedaan op basis van alle voorbeelden die we in deze Discussion hebben gezien. Als een variant veel voor komt, dan is het een algemeen nuttige variant!

### Deze varianten nemen we mee:

#### Item

**Level - enum**

- 1
- 2
- 3

**Marker - enum (design)**

- Default
- Custom

**Children - HTML (code) / Text - string (design)**

### Mis je een variant?

Laat het in deze Discussion weten als er uit gebruikersonderzoek naar voren is gekomen dat deze design beslissing een hele verbetering zou zijn.

**💡 Tip**
Je kunt [varianten ook als extensie toevoegen](https://www.nldesignsystem.nl/handboek/componenten-vinden/#ik-mis-een-functionaliteit-binnen-een-candidate-of-hall-of-fame-component-wat-nu) aan een component.

**Comment by Rozerinay** [comment]
## Candidate voorbereidingsfase: Anatomie bepaald

<img width="960" height="540" alt="Een ongeordende lijst met meerdere list items en aanwijzers. De lijst staat binnen een fel-roze gestreept kader. Een aanwijzer met het cijfer 1 wijst naar de volledige ongeordende lijst. Een aanwijzer met het cijfer 2 wijst naar een afzonderlijk list item met de tekst ‘Text’. Een aanwijzer met het cijfer 3 wijst naar de marker, weergegeven als een bolletje vóór een list item. Een aanwijzer met het cijfer 4 wijst naar de content van een list item, weergegeven als de tekst ‘Dit is een voorbeeld van een List Item in een Unordered List.’ binnen de lijst." src="https://github.com/user-attachments/assets/3926c8cb-cd91-4c10-a0f7-151f7a0b5389" />

De component bestaat uit: 

1. Unordered List: de volledige lijst.
2. List Item: een afzonderlijk item binnen de lijst.
3. Marker: voor het weergeven van een teken voorafgaand aan de content.
4. Content: voor de inhoud van het List Item.

---

[Deze schets is gemaakt in Figma](https://www.figma.com/design/0Y9Tbb373u6QGxGT6mqUnm/NL-Design-System---Doc-website---Afbeeldingen?node-id=1850-3874)

**Comment by Rozerinay** [comment]
## Candidate voorbereidingsfase: Design tokens bepaald

De design tokens voor de Candidate component zijn bepaald. Dat hebben we gedaan op basis van alle designs die we in deze Discussion hebben gezien. Als een beslissing vaker dan 2 keer voorkomt, is het een algemeen nuttige beslissing.

### Deze design tokens nemen we mee:

**General**
- `nl.unordered-list.color`
- `nl.unordered-list.font-family`
- `nl.unordered-list.font-size`
- `nl.unordered-list.line-height`
- `nl.unordered-list.margin-block-end`
- `nl.unordered-list.margin-block-start`
- `nl.unordered-list.padding-inline-start`

**Item**
- `nl.unordered-list.item.margin-block-end`
- `nl.unordered-list.item.margin-block-start`
- `nl.unordered-list.item.padding-inline-start`

**Marker**
- `nl.unordered-list.marker.color`

### Mis je een design token?

Laat het in deze Discussion weten als er uit gebruikersonderzoek naar voren is gekomen dat deze design-beslissing een hele verbetering zou zijn.

**💡 Tip**
Je kunt design tokens ook als extensie toevoegen aan de component.

- [Voor designers: Zelf componenten uitbreiden](https://www.nldesignsystem.nl/handboek/designer/zelf-componenten-uitbreiden/)
- Voor developers: Op het moment van schrijven is de documentatie hierover nog in ontwikkeling. Vraag ons er gerust naar.
